index.js

```diff
@@ -228,15 +228,14 @@ Kareem.prototype.execPostSync = function(name, context, args) {
 };
 
 Kareem.prototype.createWrapperSync = function(name, fn) {
-  var kareem = this;
-  return function syncWrapper() {
-    kareem.execPreSync(name, this, arguments);
+  return () => {
```

Comment:
My big concern is that arrow functions are always anonymous, which means this would look a bit messy in stack traces. Is there a perf benefit to your approach or is it purely stylistic?

Reply:
I suspect a Potential perf benefit but Stack traces are also Important as they could especially very messy in a Pub sub system.

Feel free to reject ;)